### ISSUE TITLE
Model weights don't load if I set layer to trainable

### ISSUE THREAD

**Issue by 1yian** [keras-team/keras#11748]
I'm trying to fine-tune a CNN by setting the convolutional part to untrainable, then when it stops improving I want to set the convolutional part to trainable then continue training.

I save the models weights (overwite=True) after training it with the convolutional part frozen, then I unfreeze the model, compile the model, then load the weights, however my loss and accuracy both reset to a very bad value.

Is there something I'm missing with the keras load functions? or is this just some glitch
